src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -511,29 +540,32 @@ protected Object clone() throws CloneNotSupportedException {
     }
 
     /**
-     * Ensures that the object referenced by the given reference remains
+     * Ensures that the given object remains
      * <a href="package-summary.html#reachability"><em>strongly reachable</em></a>,
-     * regardless of any prior actions of the program that might otherwise cause
-     * the object to become unreachable; thus, the referenced object is not
+     * regardless of any other actions of the program that might otherwise cause
+     * the object to become unreachable; thus, the object is not
```

Comment:
Delete "regardless of any other actions of the program that might otherwise cause the object to become unreachable" ? I have no idea what actions would qualify there. And it's more misleading thann informative, since IMO the real concern here is compiler dead variable elimination.

Reply:
The only action I can think of would be nulling out the last strong reference to the object in question.